driver-core/build.gradle

```diff
@@ -50,18 +50,16 @@ buildConfig {
 }
 
 jar {
-    manifest {
-        instruction 'Automatic-Module-Name', 'org.mongodb.driver.core'
-        instruction 'Build-Version', project.gitVersion
-        instruction 'Import-Package',
+    bnd(
+            '-exportcontents': "*;-noimport:=true",
+            'Automatic-Module-Name': 'org.mongodb.driver.core',
+            'Build-Version': project.gitVersion,
+            'Import-Package': [
                     'org.bson.*',                       // unfortunate that this is necessary, but if it's left out then it's not included
                     'javax.crypto.*',
-                    'javax.crypto.spec.*',
```

Comment:
Why are these packages being removed?  We do rely on 

    import javax.crypto.spec.SecretKeySpec
    import javax.naming.directory.Attribute;
    import javax.naming.directory.Attributes;
    import javax.naming.directory.InitialDirContext;
    import javax.net.ssl.SSLContext;
    import javax.net.ssl.SSLSocketFactory;

Unless these package imports are recursive by default.  I don't think they were before.

Reply:
I get the following warning when I add the removed package to the Import-Package attribute:
`warning: Unused Import-Package instructions: [javax.crypto.spec.*]`
Removing the package resolves the warning.

Reply:
I'm inclined to leave them in if it's a warning and not an error, and open another ticket to investigate it.

Reply:
I have added the packages, and I will open the ticket about the warning.